lib/msal-common/src/unifiedCache/utils/CacheHelper.ts

```diff
@@ -43,4 +45,86 @@ export class CacheHelper {
         });
         return Object.assign({}, ...keyValues);
     }
+
```

Comment:
Make sure we have tests cases for this file, especially null/empty inputs

Reply:
I added some, will append more corner cases soon